guides/v2.3/rest/bk-rest.md

```diff
@@ -13,7 +13,7 @@ functional_areas:
 The [REST API documentation][] published here on the Magento 2 Developer Documentation {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %} describes the REST APIs that are available on the latest release of Magento 2.
 
 This documentation uses [ReDoc][] to organize and present schema files that follow an [Open-API specifications][].
-The schema files this tool uses is generated from a running instance of Magento {{page.guide_version}}, and
+The schema files used by this tool are basically generated from a running instance of Magento {{page.guide_version}}, and
```

Comment:
The original version is more correct. We prefer using active voice rather than passive voice. Please revert this change.

Reply:
hi @jeff-matthews but the changes which  i have done is more understandable.But advice on this if it is necessary to revert the changes then i will revert it
Thank You

Reply:
Actually, there is a small grammar error here. It should be:

> The schema files this tool uses **are** generated from a running instance of Magento...

That's the only change we can make on this line per our style guide.